src/components/LocationPage/ChartBlock.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { Fragment } from 'react';
+import {
+  ChartHeader,
+  ChartDescription,
+  ChartLocationName,
+  BetaTag,
+  ChartHeaderWrapper,
+} from './ChartsHolder.style';
+import { Projections } from 'common/models/Projections';
+import { Projection } from 'common/models/Projection';
+import Disclaimer from 'components/Disclaimer/Disclaimer';
+import Outcomes from 'components/Outcomes/Outcomes';
+import ShareButtons from 'components/LocationPage/ShareButtons';
+import {
+  ChartRt,
+  ChartPositiveTestRate,
+  ChartICUHeadroom,
+  ChartContactTracing,
+  ChartFutureHospitalization,
+} from 'components/Charts';
+import { Metric, getMetricName } from 'common/metric';
+import { formatUtcDate } from 'common/utils';
+import { generateChartDescription } from 'common/metrics/future_projection';
+
+function Chart(props: { getDatasetData: any; metric: number; data: any }) {
+  return (
+    <Fragment>
+      {props.data && props.metric === Metric.CASE_GROWTH_RATE && (
+        <ChartRt columnData={props.getDatasetData} />
+      )}
+      {props.data && props.metric === Metric.POSITIVE_TESTS && (
+        <ChartPositiveTestRate columnData={props.data} />
+      )}
+      {props.data && props.metric === Metric.HOSPITAL_USAGE && (
+        <ChartICUHeadroom columnData={props.data} />
+      )}
+      {props.data && props.metric === Metric.CONTACT_TRACING && (
+        <ChartContactTracing columnData={props.data} />
+      )}
+    </Fragment>
+  );
+}
+
+function ChartBlock(props: {
+  chartRef: React.RefObject<HTMLDivElement>;
+  isMobile: Boolean;
+  shareButtonProps: { [key: string]: any };
+  projection: Projection;
+  metric: number;
+  datasetName?: any;
```

Comment:
changing this type to 'any' finally made typescript stop complaining..

Reply:
dropped datasetName. the data check applies to disclaimer + sharebuttons so gonna keep for now